### ISSUE TITLE
Broken links to spec/documentation in README

### ISSUE THREAD

**Issue by jmhrpr** [input-output-hk/ouroboros-network#4300]
The links on the README for the files "Introduction to the design of Data Diffusion and Networking of Cardano Shelley", "The Shelley Networking Protocol" and "The Cardano Consensus and Storage Layer" are currently 404'ing.

**Comment by jmhrpr** [comment]
Just saw this is already being addressed: https://github.com/input-output-hk/ouroboros-network/commit/165acd4c097754046e77899dd6abaced7f9ca7ac

**Comment by amesgen** [comment]
Thanks, this is a duplicate of #4209

Once #4299 is merged, the nightly GitHub Pages CI will deploy the documents to the correct place, such that the links in the README are going to work.
